app/models/user.rb

```diff
@@ -73,4 +77,4 @@ def verify_otp?(phone_number, otp_code)
       find_by(phone_number: phone_number.formatted_phone_number)
```

Comment:
The 'has_many :user_chanels' association is duplicated. Remove one of the duplicate associations to clean up the model.